Adds various carbon and human emotes

CosmicScientist opened the pull request:
Adds some emotes I either find myself upset when I can't do them or think are worthwhile and should have been in before.

CogBall committed:
Adds carbon and human emotes

Xhuis commented:
*gargle *tongue
you fool

RemieRichards commented on code/modules/mob/living/carbon/human/emote.dm:
```diff
@@ -1,6 +1,6 @@
 /mob/living/carbon/human/emote(act,m_type=1,message = null)
 	var/param = null
-
+//*grimace "<B>[src]</B>\'s" *rolleye *huff *blanch *snigger *smirk *cutthroat *gulp *tongue *grit *flounder *flush *slump *stomp *gargle *scowl
```
remove comment.

Steelpoint commented:
> flushes

What have you done.

PKPenguin321 commented:
>gulp
>tongue
>flush
>gargle

welcome to erp verbs 2.0

Ricotez commented:
might as well have added *deepthroat at this rate

Shapsy commented:
You should also add *scoff

CosmicScientist commented:
@Shapsy would that show as:
**Humie** scoffs.
?

Because if so, sure I can add it in.

phil235 commented on code/modules/mob/living/carbon/human/emote.dm:
```diff
@@ -100,6 +103,21 @@
+							break
+					if (M != src && M != null)
+						var/genderType = gender == MALE ? "his" : "her"
+						message = "<B>[src]</B> points [genderType] finger at [M] and then draws a line across [genderType] throat."
```
use \his just like below, instead of GenderType

CosmicScientist replied:
I'll test it to see if it does work but the reasoning for that was being told the second \his would pick on [M]'s gender.

CosmicScientist replied:
Testing proved it turns \his into the gender/non-gender of [M], unless I write the phrase differently, e.g. "[src] draws a line across \his throat and points at [M]", the previous line is needed.

phil235 replied:
my bad then, good catch

CosmicScientist replied:
Not my catch, it was Remie's.

Shapsy commented:
ye

CogBall committed:
Adds scoff, changes flush, ignores deepthroat

Commit:
Update human emote.dm with scoff in help

Puts scoff in the "help" text.

CosmicScientist commented:
Flush is now a little different:
**Sickly patient (405)** flushes with colour.
**Sickly patient (405)** says "Thanks doc, I feel much better now."

Since there's already blush I made it a bit more general to be an opposite to blanch.

Miauw62 commented:
>adding more emotes to the disgusting heap of shitcode that is emotes

i really need to do something about that, someday.

also more like
**Medbay stutterwhore #5897236** flushes with colour.
**Medbay stutterwhore #5897236** says, "O-oh~"

CosmicScientist commented:
@Miauw62 they can already meme with *blush, *me and the custom command. Remove those or change the player base.

Miauw62 commented:
i was making a joke :v

CosmicScientist commented:
Sorry, it's hard to tell what's a joke and what's not sometimes on here. ='(

Xhuis commented:
I still think that some of these emotes are way too erotic

CosmicScientist commented:
Which ones @Xhuis? I can solve it by adding *doubledongs so the rest seem PG13 by comparison.

Commit:
Update human emote.dm -gargle

Removes gargle because I cannot give a good reason for wanting it in and Xhuis' gutter mind.

Commit:
Update human emote.dm -helpgargle

Removes gargle from help because grawr forgot.

CosmicScientist commented:
This isn't meaningful and bloating this bit of code further isn't good.

I hope datum emotes become a thing.
